### PR TITLE
Clinical Assessment Form

### PR DESCRIPTION
**Pull request by semujju**
Clinical Assessment Form

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 1]
```diff
@@ -0,0 +1,1422 @@
...
+                            </section>
+                        </div>
+
+                        <div class="tab-pane" id="pregnancy-fp">
+                            <section>
+                             <div class="row">
+                                 <div class="col-md-6">
+
+                                    <div class="form-check form-check-inline">
+                                    <input class="form-check-input" type="checkbox" id="inlineCheckbox1" value="option1" />
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Why use Input

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 2]
```diff
@@ -0,0 +1,1422 @@
...
+                                    <label class="form-check-label" for="inlineCheckbox1">Pregnant</label>
+                                    </div>
+                                    <div class="form-check form-check-inline">
+                                      <input class="form-check-input" type="checkbox" id="inlineCheckbox2" value="option2" />
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: WHy Use input

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 3]
```diff
@@ -0,0 +1,1422 @@
...
+                                      <label class="form-check-label" for="inlineCheckbox2">Breast Feeding</label>
+                                    </div>
+                                    <div class="form-check form-check-inline">
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Why Use Input

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 4]
```diff
@@ -0,0 +1,1422 @@
...
+                                      <input class="form-check-input" type="checkbox" id="inlineCheckbox3" value="option3" />
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Why Use Input

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 5]
```diff
@@ -0,0 +1,1422 @@
...
+                                      <label class="form-check-label" for="inlineCheckbox3">Not pregnant/Not Breast feeding</label>
+                                    </div>
+                                    <div class="form-check form-check-inline">
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Why Use Input

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 6]
```diff
@@ -0,0 +1,1422 @@
...
+                                      <input class="form-check-input" type="checkbox" id="inlineCheckbox3" value="option3" />
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Why use Input

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 7]
```diff
@@ -0,0 +1,1422 @@
...
+                                            </span>
+                                        </span>
+                                        </div>
+                                    </div> -->
+                                 <br/>
+                                <div class="row">
+                                    <div class="col-md-6">
+                                        <label>Is on family Planning?</label>
+                                    <div class="form-check form-check-inline">
+                                      <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio1" value="option1" />
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Why use Input

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 8]
```diff
@@ -0,0 +1,1422 @@
...
+                                      <label class="form-check-label" for="inlineRadio1">No</label>
+                                    </div>
+                                    <div class="form-check form-check-inline">
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Why use input What happened to the <obs> tag

**Review by slubwama — CHANGES_REQUESTED** [review]

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml (hunk 8 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: They are place holders for now, this is still WIP

**Pull request closed without merging**